src/sentry/static/sentry/app/views/organizationIntegrations/abstractIntegrationDetailedView.tsx

```diff
@@ -300,11 +347,15 @@ class AbstractIntegrationDetailedView<
                 <AuthorName>{this.author}</AuthorName>
               </div>
             )}
-            {this.resourceLinks.map(({title, url}) => (
-              <ExternalLink key={url} href={url}>
-                {t(title)}
-              </ExternalLink>
-            ))}
+            {this.resourceLinks.map(({title, url}) => {
+              const icon = title === 'View Source' ? <IconCode /> : <StyledIconGithub />;
```

Comment:
@NisanthanNanthakumar Not every icon is going to be a link to link to the source code or Github. For example, Twilio has a link to Twilio. We should have a third case with some default icon if it's something other than "View Source" or "Report Issue".

Reply:
@scefali any suggestions? For `Documentation`, I think `IconDocs` would look good. For the logos, I think we can make a  switch case function to give appropriate logo icons. Default brand icon would be `IconGeneric`. Icons and logos can be seen [here](https://www.figma.com/file/Gq0SG3GrwBu3S1NArPjB5bEo/Sentry-Design-System?node-id=745%3A1278)

Reply:
@NisanthanNanthakumar yea, sounds good to me